src/main/java/com/google/codeu/data/Datastore.java

```diff
@@ -55,16 +56,17 @@ public void storeMessage(Message message) {
    *     message. List is sorted by time descending.
    */
   public List<Message> getMessages(String user) {
+    List<Message> messages;
     Query query =
         new Query("Message")
             .setFilter(new Query.FilterPredicate("user", FilterOperator.EQUAL, user))
             .addSort("timestamp", SortDirection.DESCENDING);
-
     messages = getMessagesHelperFunction(query);
     return messages;
   }
 
   public List<Message> getAllMessages(){
```

Comment:
Sorry that I'm only looking at the code at this late stage, but I'd recommend adding a method description here, like getMessages above.

getAllMessages is a public method and so having a method description is important so that users know how to use this public method.
```suggestion
  /**
   * Gets all messages.
   *
   * @return a list of all messages posted, or empty list if no messages have
   *     been posted. List is sorted by time descending.
   */
  public List<Message> getAllMessages(){
```